conans/test/functional/file_copier_test.py

```diff
@@ -1,5 +1,7 @@
 import unittest
 import os
+from turtledemo.chaos import f
```

Comment:
I have no idea how that import got there ...
probably did some fancy key shortcut without noticing